gpu_bdb/benchmark_runner/benchmark_config.yaml

```
@@ -1,21 +1,21 @@
 # benchmark config yaml
 ### Please fill these accordingly
-data_dir:
+data_dir: 
 output_dir: ./
 file_format: parquet
 output_filetype: parquet
 split_row_groups: False
 repartition_small_table: True
 benchmark_runner_include_bsql: False
 
-scheduler_file_path:
+scheduler_file_path: 
 dask_profile: False
 dask_dir: ./
-num_workers: 16
+num_workers: 8
```

If we go back to the original memory limit, this should return to 16. If not, reject this suggestion,
```suggestion
num_workers: 16
```

---

@ChrisJar what was the memory limit used w/ 80GB A100s?

---

I used a pool size of 80GB and a device memory limit of 40GB